tests/benchdnn/graph/bench_graph.cpp

```diff
@@ -36,18 +36,27 @@ void check_correctness(const settings_t &s) {
     for (const auto &i_mb : s.mb) {
         deserialized_graph_t dg;
         dg.load(locate_file(s.json_file));
-        flex_rewrite_t fw(i_in_shapes, i_op_attrs, i_fpmath_mode, i_mb, i_dt,
-                i_dt_map, i_op_kind_map);
-        fw.rewrite(dg);
-        BENCHDNN_PRINT(7, "[INFO] Graph dump:\n%s\n", dg.get_string().c_str());
 
-        const prb_t prb(dg, i_expected_n_partition);
+        res_t res {};
         const auto &cpp_pstr = case_to_str(s.json_file, i_in_shapes, i_op_attrs,
                 i_fpmath_mode, i_expected_n_partition, i_mb, i_dt, i_dt_map,
                 i_op_kind_map);
         const char *pstr = cpp_pstr.c_str();
+
+        // rewrite the graph
+        flex_rewrite_t fw(i_in_shapes, i_op_attrs, i_fpmath_mode, i_mb, i_dt,
+                i_dt_map, i_op_kind_map);
+        if (fw.rewrite(dg) != OK) {
+            res.state = SKIPPED;
+            res.reason = skip_reason::invalid_case;
+            parse_result(res, pstr);
+
+            continue;
+        }
```

Comment:
I don't share the vision that unsuccessful rewriting leads to a valid case that won't be visible from an automated run.

Another thing is that parser errors abort execution for rest drivers for the reason that user's intention must be clear (`it's about `continue`).

Rest part LGTM.